Added new step for /var/lib/pgsql as a prerequisite (#1186)
theforeman/foreman-documentation: guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_prerequisites.adoc

To confirm if a template has been edited, you can view its *History* before you upgrade or view the changes in the audit log after an upgrade.
In the {ProjectWebUI}, navigate to *Monitor* > *Audits* and search for the template to see a record of changes made.
If you use the export method, restore your changes by comparing the exported template and the default template, manually applying your changes.
* If you previously upgraded {Project} from an earlier version, and the `/var/lib/pgsql` contained the PostgreSQL database content before the migration from PostgreSQL 9 to PostgreSQL 12 from the SCL, empty `/var/lib/pgsql` before proceeding.

> * {Project} {ProjectVersionPrevious}'s migrated `postgres` database content is stored in this directory.

Reviewer: This line can be removed now, as the content was merged into the previous sentence.

Author: Done and committed.

To confirm if a template has been edited, you can view its *History* before you upgrade or view the changes in the audit log after an upgrade.
In the {ProjectWebUI}, navigate to *Monitor* > *Audits* and search for the template to see a record of changes made.
If you use the export method, restore your changes by comparing the exported template and the default template, manually applying your changes.

> ifdef::satellite[]

Reviewer: It's not specific to Satellite.

> * If you previously upgraded {Project} from an earlier version, or if the `/var/lib/pgsql` directory has any content, ensure it is empty before proceeding.

Reviewer 2: `or` feels odd to me. Isn't it `and`? Or rather: if `/var/lib/pgsql` has any content (as is the case when upgrading from older versions), empty it.

> * If you previously upgraded {Project} from an earlier version, and the `/var/lib/pgsql` directory contains any content, as it does when upgrading from older versions, empty it before proceeding.

Author: Made the change and will re commit. Thank you!

> * {Project} {ProjectVersionPrevious}'s migrated `postgres` database content is stored in this directory.

Reviewer: This reads as if `/var/lib/pgsql` contains the *current* database content, which it does not. It does contain some old, not-cleaned-up content. I'd probably either completely remove this sentence, or reword it to something like: `/var/lib/pgsql` contained the PostgreSQL database content before the migration from PostgreSQL 9 to PostgreSQL 12 from the SCL.

To confirm if a template has been edited, you can view its *History* before you upgrade or view the changes in the audit log after an upgrade.
In the {ProjectWebUI}, navigate to *Monitor* > *Audits* and search for the template to see a record of changes made.
If you use the export method, restore your changes by comparing the exported template and the default template, manually applying your changes.
ifdef::satellite[]
* If you previously upgraded {Project} from an earlier version, or if the `/var/lib/pgsql` directory has any content, ensure it is empty before proceeding.
endif::[]
ifdef::satellite[]

> * {Project} {ProjectVersionPrevious}'s migrated `postgres` database content will be stored in this directory.

Reviewer 3: ```suggestion
* {Project} {ProjectVersionPrevious}'s migrated `postgres` database content is stored in this directory.
```
Use the present tense at all times, as per IBM style.

To confirm if a template has been edited, you can view its *History* before you upgrade or view the changes in the audit log after an upgrade.
In the {ProjectWebUI}, navigate to *Monitor* > *Audits* and search for the template to see a record of changes made.
If you use the export method, restore your changes by comparing the exported template and the default template, manually applying your changes.

> * If you previously upgraded {Project} from 6.7 or an earlier version, or if the `/var/lib/pgsql` directory has any content, ensure it is empty before proceeding. {Project} {ProjectVersionPrevious}'s migrated `postgres` database content will be stored in this directory.

Reviewer 4: `6.7` is Satellite-only. Please check if this file is only included in builds for Satellite. If it also shows for Foreman, please either remove the version number or encapsulate in within `ifdef::satellite[]` and add an appropriate sentence for Foreman please.

Also, please stick to one sentence per line.
